skbio/sequence/_rna.py

```diff
@@ -135,6 +135,73 @@ def degenerate_map(cls):
     def _motifs(self):
         return _motifs
 
+    @stable(as_of="0.4.0-dev")
+    def reverse_transcribe(self):
+        """Reverse transcribe RNA into DNA.
+
+        RNA sequence is assumed to be the coding strand. Uracil (U) is
```

Comment:
cool. I will update docstrings for both transcribe() and
reverser_transcribe()

On Mon, Aug 17, 2015 at 3:02 PM, Evan Bolyen <notifications@github.com>
wrote:

> In skbio/sequence/_rna.py
> <https://github.com/biocore/scikit-bio/pull/1057#discussion_r37242330>:
>
> > @@ -135,6 +135,73 @@ def degenerate_map(cls):
> >      def _motifs(self):
> >          return _motifs
> >
> > +    @stable(as_of="0.4.0-dev")
> > +    def reverse_transcribe(self):
> > +        """Reverse transcribe RNA into DNA.
> > +
> > +        RNA sequence is assumed to be the coding strand. Uracil (U) is
>
> [image: :+1:] [image: :heavy_multiplication_x:] [image: :100:] for sense
> and antisense that is much more precise.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio/pull/1057/files#r37242330>.
>